.travis.yml

```diff
@@ -66,7 +66,7 @@ matrix:
         - npm run coveralls
 
     - language: node_js
-      node_js: 7
+      node_js: node
```

Comment:
Ref: <https://github.com/creationix/nvm/blob/master/README.md#usage> `node` version means the latest version of node.js